src/main/java/it/pn/frontend/e2e/pages/destinatario/personaFisica/NotifichePFPage.java

```diff
@@ -312,4 +318,24 @@ public void waitLoadNotificheDEPageDelegante(String nome, String cognome) {
         }
 
     }
+
+    public void clickFiltraButton() {
+        getWebDriverWait(30).withMessage("Il bottone filtra nella pagina ricerca Notifiche PF non è cliccabile").until(ExpectedConditions.elementToBeClickable(this.filtraButton));
+        this.filtraButton.click();
+    }
+
+    public boolean isErrorMessageDisplayed() {
+        return getWebDriverWait(30).withMessage("Il messagio di errore non e visibile").until(ExpectedConditions.visibilityOf(nonValidIunMessage)).isDisplayed();
```

Comment:
scrivere messaggio e non ' messagio'. aggiusta la variabile nonValidIunMessage in notValidIunMessage

Reply:
sto cambiando , grazie